yb-voyager/cmd/analyzeSchema.go

```diff
@@ -452,6 +454,25 @@ func checkDDL(sqlInfoArr []sqlInfo, fpath string) {
 			reportCase(fpath, "LANGUAGE C not supported yet.",
 				"", "", "FUNCTION", tbl[2], sqlInfo.formattedStmt)
 			summaryMap["FUNCTION"].invalidCount++
+		} else if regMatch := partitionColumnsRegex.FindStringSubmatch(sqlInfo.stmt); regMatch !=nil {
+			allColumns := strings.Trim(regMatch[3], `()`)
+			allColumns = strings.Trim(allColumns, " ")
+			allColumnsList := strings.Split(allColumns, "(")
+
+			primaryKeyColumns := allColumnsList[len(allColumnsList)-1]
+			primaryKeyColumns = strings.Trim(primaryKeyColumns, `()`)
```

Comment:
is there a need for this line after the trim on line 458?

Reply:
Yes, it is required as one `)` closing bracket will be there at the end when I splited the allColumns with respect to `(`.

Reply:
Doesn't `allColumns = strings.Trim(allColumns, "() ")` remove all trailing and leading instances of `(`, `)` and whitespaces? We seem to be picking up only the last element from `allColumnsList`, which should have all leading instances (as you mentioned in your case, `)`) removed

Reply:
yes it should, made the changes. Please review it again. Thanks